src/rpcmining.cpp

```diff
@@ -47,7 +46,6 @@ Value getmininginfo(const Array& params, bool fHelp)
     uint64_t nWeight = 0;
     int64_t nTime= GetAdjustedTime();
     pwalletMain->GetStakeWeight(nWeight);
-    uint64_t nNetworkWeight = GetPoSKernelPS();
```

Comment:
Why you deleted the variable? The value is used in two places and GetPoSKernelPS is fairly expensive.

Reply:
Because NetworkWeight2 was double and thats been removed. NetworkWeight was uint64_t.

startails code is double for stats site. figure it'd better just to show actual full network weight with decimal and all in getmininginfo much like NetworkWeight2 did. he also collects stats there a lot.

it is set uint64_t nNetworkWeight in bitcoingui.cpp for staking info in ui (green arrow for staking).
the ui main page for Network Weight ends up RoundToString(GetPoSKernelPS(),2)
also i believe getmininginfo should have the full double or a rounded. 

again unless we get startail to change a code on the stats page I dont see a point of leaving a uint64_t value in getmininginfo. thats just cosmetic for wallet.

all other Network Weight functions as originally did in UI both places and in getmininginfo it functions the same as NetworkWeight2